Week1/Solution Week-1/Solution-6.py

```diff
@@ -0,0 +1,11 @@
+#6.Write a python script to enter any string and print only part of string. Take input of start character and end character from user.
+
+
+def partstring(s):
+    st=int(input("Enter start position:"))
+    e=int(input("Enter end position:"))
+    
+    print(f"The part of the string '{s}' is '{s[st-1:e:]}'")

```

Comment:
Can you explain this statement...?

Reply:
here i use f-string and string slicing. meaning of st-1 is first letter of string we use 0-based indexing in programming but the user you give us input may be don't aware from it so for them the 1st index is 1st but for us we use 0th index and in this we use 0th index to end index that denote by 'e'. i hope you get the point.thank you for commenting.

Reply:
Yes.. 
Thank you for your reply

Reply:
why use this code?

Reply:
how logic is this